projectq/setups/decompositions/phaseestimation_test.py

```diff
@@ -179,7 +178,7 @@ def test_X_no_eigenvectors():
     assert total == pytest.approx(N, abs=5)
     assert plus_probability == pytest.approx(
         1.0 / 4.0, abs=1e-1
-    ), "Statistics on |+> probability are not correct (%f vs. %f)" % (
+    ), "Statistics on |+> probability are not correct ({:f} vs. {:f})".format(
```

Comment:
Why not use the `f'{total_prob_after:f}` formatting here?

Reply:
Will be addressed in the next commit.